lib/dry/monads.rb

```diff
@@ -39,5 +40,17 @@ def Right(value)
     def Left(value)
       Either::Left.new(value)
     end
+
+    # @param value [Object] the value to be stored in the monad
+    # @return [Result::Success]
+    def Success(value)
+      Result::Success.new(value)
+    end
+
+    # @param value [Object] the value to be stored in the monad
+    # @return [Result::Failure]
+    def Failire(value)
```

Comment:
I guess it should be `Failure` not `Failire`, shouldn't it?

Reply:
It should indeed, thanks 😃